src/providers/sms/seven.js

```diff
@@ -0,0 +1,48 @@
+/* @flow */
+import fetch from '../../util/request'
+// Types
+import type { SmsRequestType } from '../../models/notification-request'
+
+export default class SmsSevenProvider {
+  id: string = 'sms-seven-provider'
+  apiKey: string
+
+  constructor ({ apiKey }: Object) {
+    this.apiKey = apiKey
+  }
+
+  /*
+   * Note: 'nature' is not supported.
+   */
+  async send (request: SmsRequestType): Promise<string> {
+    const { from, text, to, type, ttl, messageClass } = request.customize ? (await request.customize(this.id, request)) : request
+    const params = {
+      flash: messageClass === 0,
+      from,
+      text,
+      to,
+      ttl,
+      unicode: type === 'unicode'
+    }
```

Comment:
It seems they use `0` or `1` in place of booleans in [their documentation](https://www.seven.io/en/docs/gateway/http-api/sms-dispatch/#optional-parameters):
```suggestion
    const params = {
      flash: messageClass === 0,
      from,
      text,
      to,
      ttl,
      unicode: type === 'unicode'
    }
```
```suggestion
    const params = {
      flash: messageClass === 0 ? 1 : 0,
      from,
      text,
      to,
      ttl,
      unicode: type === 'unicode' ? 1 : 0
    }
```

Reply:
Both works :)